src/scripts/farming/BerryType.ts

```diff
@@ -1,11 +1,85 @@
 enum BerryType {
     None = -1,
+
+    // First generation
     Cheri,
     Chesto,
     Pecha,
     Rawst,
     Aspear,
     Leppa,
     Oran,
-    Sitrus
+    Sitrus,
+
+    // Second generation
+    Persim,
+    Razz,
+    Bluk,
+    Nanab,
+    Wepear,
+    Pinap,
+
+    Figy,
+    Wiki,
+    Mago,
+    Aguav,
+    Iapapa,
+
+    Lum,
```

Comment:
Is there any reason for all the extra spacings in this file?
```suggestion
    Pinap,
    Figy,
    Wiki,
    Mago,
    Aguav,
    Iapapa,
    Lum,
```

Reply:
I separated them into groups for determining properties. The spaces can be removed.